build/charts/theia/values.yaml

```diff
@@ -0,0 +1,27 @@
+clickhouse:
+  # -- Container image to use for the ClickHouse Monitor.
+  monitorImage:
+    repository: "projects.registry.vmware.com/antrea/theia-clickhouse-monitor"
+    pullPolicy: "IfNotPresent"
+    tag: "latest"
+  # -- ClickHouse username. It will be stored in a secret.
+  username: "clickhouse_operator"
+  # -- ClickHouse password. It will be stored in a secret.
+  password: "clickhouse_operator_password"
+  # -- HTTP port number for the ClickHouse service.
+  httpPort: 8123
+  # -- TCP port number for the ClickHouse service.
+  tcpPort: 9000
+  # -- ClickHouse storage size. Can be a plain integer or as a fixed-point
+  # number using one of these quantity suffixes: E, P, T, G, M, K. Or the
+  # power-of-two equivalents: Ei, Pi, Ti, Gi, Mi, Ki.
+  storageSize: "8Gi"
```

Comment:
I was expecting much more customization here (with PV support, etc.)

Reply:
I have the plan to have more customization like PV related options and monitor customizations including whether to enable it and some parameters used in the monitor. But I'm not sure whether I should include all of these in this PR or it would be clear to add them in follow-up PRs?